hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/selective/BalancingJob.java

```diff
@@ -51,12 +51,12 @@ public void run() {
         try {
             if (consumersRegistry.isLeader()) {
                 try (Timer.Context ctx = metrics.consumersWorkloadRebalanceDurationTimer(kafkaCluster).time()) {
-                    logger.info("Initializing workload balance.");
+//                    logger.info("Initializing workload balance.");
```

Comment:
debug leftover. As far as I know workload balance in integration tests runs very often (few times per second). Maybe we don't need to run it so often. If we do then maybe we should disable logger for this class in integration tests.